src/model/ProductReviewsModel.ts

```diff
@@ -0,0 +1,76 @@
+import { db } from "@vercel/postgres";
+
+export type ProductReview = {
+  id: string;
+  user_id: string;
+  product_id: string;
+  product_name: string;
+  review: string;
+  rating: string;
+  gender: string;
+  age: string;
+  skintype: string;
+  added_at: string;
+  title: string;
+  recomanding: boolean;
```

Comment:
For now only the fields inside the DB should be part of this interface. We'll add to it later, when we integrate with the user.